community/portworx/templates/lighthouse-service.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     tier: px-web-console
 {{- include "px.labels" . | nindent 4 }}
 spec:
-  type: NodePort
+  type: LoadBalancer
```

Comment:
In case of IBM, try putting ClusterIP so that the end user can add an ingress controller to it, to access it via the service endpoints that IKS gives.

Reply:
LoadBalancer actually creates an external load balancer IP that can be accessed from outside the cluster